### PR TITLE
 First implementation of the Rao-Sandelius permutation algorithm on column

### PR DESCRIPTION
**Pull request by giupardeb**
this is only the first implementation, I will work to improve the code further.

### CODE REVIEW

**Comment by prasser** [comment]
Thanks! Please take a look at the changes in your prior pull request (see https://github.com/arx-deidentifier/arx/pull/247). The function needs to be added in some further places and a MaskingFunctionDescription needs to be provided. Best, Fabian

**Comment by giupardeb** [comment]
> Thanks! Please take a look at the changes in your prior pull request (see #247). The function needs to be added in some further places and a MaskingFunctionDescription needs to be provided. Best, Fabian

Thank you! I thought I did it..but I recheck better!

**Comment by giupardeb** [comment]
I think that I have fixed all issues, but if I don't, please tell me and I will do all necessary things to fix it. Thanks!

File: src/main/org/deidentifier/arx/masking/DataMaskingFunction.java [hunk 1]
```diff
@@ -161,9 +161,14 @@ private void raoSandelius(DataColumn column) {
 			
 			for (int i=0;i<lengthColumn;i++)
 				column.set(i, colout.get(i));
-			
 		}
 		
+		/**
+		 *
```
Comment (prasser) [inline src/main/org/deidentifier/arx/masking/DataMaskingFunction.java]: Please provide a description here

Comment (giupardeb) [reply src/main/org/deidentifier/arx/masking/DataMaskingFunction.java]: Done! :-)

**Pull request merged**